libraries/stdlib/jvm/src/kotlin/io/files/FileTreeWalk.kt

```diff
@@ -12,6 +12,20 @@ import java.io.File
 import java.io.IOException
 import java.util.ArrayDeque
 
+/**
+ * An enumeration to describe possible algorithms to traverse.
+ * There are two of them: search in breadth (same directory depth)
+ */
+public enum class WalkAlgorithm {
+    /**
+     * Breadth-first search
+     */
+    BFS,
```

Comment:
I actually believe we should write these out as `BREADTH_FIRST` and `DEPTH_FIRST`, just like we have `TOP_DOWN` instead of `TD` below.

Reply:
Agree.

Reply:
> I actually believe we should write these out as `BREADTH_FIRST` and `DEPTH_FIRST`, just like we have `TOP_DOWN` instead of `TD` below.

Done.